### PR TITLE
Removed Burrow from toolchain

### PR DESCRIPTION
**Pull request by johannwagner**

### CODE REVIEW

**Comment by martin31821** [comment]
LGTM.

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,15 @@
+module github.com/EmbeddedEnterprises/autobahnkreuz
+
+go 1.12
```
Comment (fin-ger) [inline go.mod]: Why is go 1.12 needed?

Comment (johannwagner) [reply go.mod]: I did not touch this file. This is the file, which was generated from `burrow migrate`. My local version is a 1.12, but this should not matter.

Comment (fin-ger) [reply go.mod]: Okay, could you check if go 1.12 is really needed?

**Pull request merged**